test/e2e/adminUI/tests/group005Fields/color/uxTestColorField.js

```diff
@@ -0,0 +1,83 @@
+module.exports = {
+	before: function (browser) {
+		browser.app = browser.page.app();
+		browser.signinPage = browser.page.signin();
+		browser.listPage = browser.page.list();
+		browser.itemPage = browser.page.item();
+		browser.initialFormPage = browser.page.initialForm();
+
+		browser.app.navigate();
+		browser.app.waitForElementVisible('@signinScreen');
+
+		browser.signinPage.signin();
+		browser.app.waitForElementVisible('@homeScreen');
+	},
+	after: function (browser) {
+		browser.app.signout();
+		browser.end();
+	},
+	'Color field can be filled via the initial modal': function (browser) {
+		browser.app
+			.click('@fieldsMenu')
+			.waitForElementVisible('@listScreen')
+			.click('@colorsFieldsSubmenu')
+			.waitForElementVisible('@listScreen');
+
+		browser.listPage
+			.click('@createFirstItemButton');
+
+		browser.app
+			.waitForElementVisible('@initialFormScreen');
+
+		browser.initialFormPage.section.form.section.colorList.section.name
+			.fillInput({value: 'Name Field Test 1'});
```

Comment:
Now that I look at these inputs more...we should either call it something like "Color Field Test 1" or simply "Test 1"...most likely all field tests have been doing the same :-)